src/BuildingRegistry.Consumer.Address/Projections/CommandHandlingKafkaProjection.cs

```diff
@@ -132,6 +132,15 @@ public CommandHandlingKafkaProjection(IDbContextFactory<BackOfficeContext> backO
                     ct);
             });
 
+            When<AddressWasRemovedBecauseStreetNameWasRemoved>(async (commandHandler, message, ct) =>
```

Comment:
Also consume in src/BuildingRegistry.Consumer.Address/Projections/AddressKafkaProjection.cs + test test/BuildingRegistry.Tests/ProjectionTests/Consumer.Address/AddressConsumerKafkaProjectionTests.cs